src/routes/ChatList.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { HeaderContains, HeaderText } from '../components/layout/CommonStyle';
+import UnderNavBar from '../components/layout/UnderNavBar';
+import Messages from '../components/ChatList/Chattings';
+import message from '../data/message.json';
+import { BsChatDots } from 'react-icons/bs';
+
+const ChatList = () => {
+  return (
+    <ChatListContainer>
+      <HeaderContains>
+        <BsChatDots size={20} />
+        <HeaderText>Message</HeaderText>
+      </HeaderContains>
+      <Content>
+        {message.map((userMessage) => (
+          <Messages
+            key={userMessage.userId}
+            userId={userMessage.userId}
+            userName={userMessage.userName}
+            message={userMessage.messages[userMessage.messages.length - 1].text}
+          />
+        ))}
+      </Content>
+      <UnderNavBar />
+    </ChatListContainer>
```

Comment:
아래와같이 작성하셔도 똑같이 보인답니다!

```suggestion
    <>
      <HeaderContains>
        <BsChatDots size={20} />
        <HeaderText>Message</HeaderText>
      </HeaderContains>

        {message.map((userMessage) => (
          <Messages
            key={userMessage.userId}
            userId={userMessage.userId}
            userName={userMessage.userName}
            message={userMessage.messages[userMessage.messages.length - 1].text}
          />
        ))}

      <UnderNavBar />
    </>
```

Reply:
쓸데없는 스타일 컴포넌트만 잔뜩 만들었었군요... 짚어주셔서 감사합니당...